Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.12
+
+COPY scrapedumper scrapedumper
+
+CMD ["./scrapedumper"]
```

Comment:
This will use the default config, so anyone running it from the base image will need to provide a few envars to configure the dumpers. You can also use it as a base image (which is what I'm doing separately), `COPY` on a config file and override the `CMD` to use it.